ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3412,8 +3415,12 @@ signal_list_t* create_operation_node(ast_node_t* op, signal_list_t** input_lists
         case ASL: // <<<
         case SL:  // <<
             /* Shifts doesn't matter about port size, but second input needs to be a number */
-            //output_port_width = input_lists[0]->count + (shift_left_value_with_overflow_check(0x1, input_lists[1]->count)-1);
-            output_port_width = input_lists[0]->count + (shift_left_value_with_overflow_check(0x1, log2(op->children[1]->types.vnumber->get_value()), op->loc));
+            // TEMPERORY ASSIGNMENT USING IF EXPRESSION
+            if (op->children[1]->type != NUMBERS)
+                output_port_width = input_lists[0]->count + (shift_left_value_with_overflow_check(0x1, input_lists[1]->count, op->loc)-1);
```

Comment:
Output size is dictated by the output width, not the input. Again

Reply:
Output size is specified here, there is nothing about output_list or the module output width!